packages/lesswrong/components/posts/PostsItem2.tsx

```diff
@@ -365,16 +343,31 @@ const PostsItem2 = ({
   translucentBackground=false,
   forceSticky=false
 }: {
+  /** post: The post displayed.*/
   post: PostsList,
+  /** tagRel: (Optional) The relationship between this post and a tag. If
+  /* provided, UI will be shown with the score and voting on this post's
+  /* relevance to that tag.*/
   tagRel?: WithVoteTagRel|null,
+  /** defaultToShowComments: (bool) If set, comments will be expanded by default.*/
   defaultToShowComments?: boolean,
+  /** sequenceId, chapter: If set, these will be used for making a nicer URL.*/
   sequenceId?: string,
   chapter?: any,
+  /** index: If this is part of a list of PostsItems, its index (starting from
+   /* zero) into that list. Used for special casing some styling at start of
+   /* the list.*/
   index?: number,
+  /** terms: If this is part of a list generated from a query, the terms of that
+  /* query. Used for figuring out which sticky icons to apply, if any.*/
```

Comment:
```suggestion
  /**
   * terms: If this is part of a list generated from a query, the terms of that
   * query. Used for figuring out which sticky icons to apply, if any.
   */
```

Jim would like to be on on record that this is the idiomatic style for the codespace.

Reply:
But no need to change them all

Reply:
Noted!